src/polling.ts

```diff
@@ -20,6 +20,7 @@ export class PollingProcessor {
   private eventBus: EventEmitter;
   private timeout: NodeJS.Timeout;
   private log: ConsoleLog;
+  private lastPollTime  = 0;
```

Comment:
Looks like there's an extra space here. Remember to use `prettier` on save for these kinda things.

Reply:
I'll run npm format here and check in separately (as to not cause too much noise in the diff)